### ISSUE TITLE
PyPDF2.errors.DeprecationError prevents clue report print

### ISSUE THREAD

**Issue by caver456** [ncssar/radiolog#640]
Clue dialog stays open; this prevents print and prevents accept of the clue dialog:

```
070450:generating clue report pdf: C:\Users\caver\RadioLog\New_Incident_2023_03_09_070423\New_Incident_2023_03_09_070423_clue01.pdf
070451:parsed instructions:['asdfasdf']
Traceback (most recent call last):
  File "radiolog.py", line 6663, in accept
  File "radiolog.py", line 3092, in printClueReport
  File "PyPDF2\_reader.py", line 1974, in __init__
  File "PyPDF2\_utils.py", line 369, in deprecation_with_replacement
  File "PyPDF2\_utils.py", line 351, in deprecation
PyPDF2.errors.DeprecationError: PdfFileReader is deprecated and was removed in PyPDF2 3.0.0. Use PdfReader instead.
```

**Comment by caver456** [comment]
next:

```
080518:generating clue report pdf: C:\Users\caver\RadioLog\New_Incident_2023_03_09_080505\New_Incident_2023_03_09_080505_clue01.pdf
080518:parsed instructions:['asdf']
Traceback (most recent call last):
  File "C:\Users\caver\Documents\GitHub\radiolog\radiolog.py", line 6835, in accept
    self.parent.parent.printClueReport(clueData)
  File "C:\Users\caver\Documents\GitHub\radiolog\radiolog.py", line 3196, in printClueReport
    templatePage.mergePage(overlayPDF.pages[0])
  File "C:\Users\caver\AppData\Roaming\Python\Python310-32\site-packages\PyPDF2\_page.py", line 693, in mergePage
    deprecation_with_replacement("mergePage", "merge_page", "3.0.0")
  File "C:\Users\caver\AppData\Roaming\Python\Python310-32\site-packages\PyPDF2\_utils.py", line 369, in deprecation_with_replacement
    deprecation(DEPR_MSG_HAPPENED.format(old_name, removed_in, new_name))
  File "C:\Users\caver\AppData\Roaming\Python\Python310-32\site-packages\PyPDF2\_utils.py", line 351, in deprecation
    raise DeprecationError(msg)
PyPDF2.errors.DeprecationError: mergePage is deprecated and was removed in PyPDF2 3.0.0. Use merge_page instead.
```

**Comment by caver456** [comment]
next:

```
080658:generating clue report pdf: C:\Users\caver\RadioLog\New_Incident_2023_03_09_080644\New_Incident_2023_03_09_080644_clue01.pdf
080658:parsed instructions:['sadf']
Traceback (most recent call last):
  File "C:\Users\caver\Documents\GitHub\radiolog\radiolog.py", line 6784, in keyPressEvent
    self.accept()
  File "C:\Users\caver\Documents\GitHub\radiolog\radiolog.py", line 6835, in accept
    self.parent.parent.printClueReport(clueData)
  File "C:\Users\caver\Documents\GitHub\radiolog\radiolog.py", line 3197, in printClueReport
    outputPDF=PdfFileWriter()
  File "C:\Users\caver\AppData\Roaming\Python\Python310-32\site-packages\PyPDF2\_writer.py", line 2821, in __init__
    deprecation_with_replacement("PdfFileWriter", "PdfWriter", "3.0.0")
  File "C:\Users\caver\AppData\Roaming\Python\Python310-32\site-packages\PyPDF2\_utils.py", line 369, in deprecation_with_replacement
    deprecation(DEPR_MSG_HAPPENED.format(old_name, removed_in, new_name))
  File "C:\Users\caver\AppData\Roaming\Python\Python310-32\site-packages\PyPDF2\_utils.py", line 351, in deprecation
    raise DeprecationError(msg)
PyPDF2.errors.DeprecationError: PdfFileWriter is deprecated and was removed in PyPDF2 3.0.0. Use PdfWriter instead.
```

**Comment by caver456** [comment]
next:

PyPDF2.errors.DeprecationError: addPage is deprecated and was removed in PyPDF2 3.0.0. Use add_page instead.

Ok now it seems to be good.  This is definitely worth a hotfix.  Will think about whether it should be 3.5.1 or 3.6.0, based on other fixes that have happened on master since 3.5.0, and will install at 91 tonight.
